app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -156,4 +173,47 @@ private String getKeyHash(){
             return "";
         }
     }
+
+    private void deleteExistingApks() {
```

Comment:
This method can be removed after merging the latest changes from the `master` branch.

Reply:
Just committed the changes. Kindly take a look.